### PR TITLE
Add $params to files from pull request

### PR DESCRIPTION
**Pull request by kubawerlos**
Example:

this PR: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3475

changes: 229 files
here: https://api.github.com/repos/FriendsOfPHP/PHP-CS-Fixer/pulls/3475/files we can see 30 files
here: https://api.github.com/repos/FriendsOfPHP/PHP-CS-Fixer/pulls/3475/files?per_page=100 we can see 100 files
here: https://api.github.com/repos/FriendsOfPHP/PHP-CS-Fixer/pulls/3475/files?page=3&per_page=100 we can see 29 files

### CODE REVIEW

**Comment by kubawerlos** [comment]
@acrobat I cannot see it

File: lib/Github/Api/PullRequest.php [hunk 1]
```diff
@@ -90,9 +90,14 @@ public function commits($username, $repository, $id)
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/commits');
     }
 
-    public function files($username, $repository, $id)
+    public function files($username, $repository, $id, array $params = [])
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/files');
+        $parameters = array_merge([
+            'page' => 1,
```
Comment (acrobat) [inline lib/Github/Api/PullRequest.php]: I don't think we should set the default values, let github decide on them. Just allow users to override these parameters

**Comment by acrobat** [comment]
@kubawerlos the comment was somehow lost, I've re-added it!

File: lib/Github/Api/PullRequest.php (hunk 1 above)
Comment (kubawerlos) [reply lib/Github/Api/PullRequest.php]: @acrobat sounds fair, done

**Comment by kubawerlos** [comment]
ping @acrobat

**Comment by acrobat** [comment]
Thanks @kubawerlos! And congrats on your first contribution! 🎉

**Comment by kubawerlos** [comment]
@acrobat thanks. Why not to continue then -> https://github.com/KnpLabs/php-github-api/pull/817

**Pull request merged**